pom.xml

```diff
@@ -328,6 +329,12 @@
       <artifactId>reflections</artifactId>
       <version>0.10.2</version>
     </dependency>
+    <dependency>
+      <groupId>jakarta.annotation</groupId>
+      <artifactId>jakarta.annotation-api</artifactId>
+      <version>1.3.5</version>
+      <scope>provided</scope>
```

Comment:
Please, why is it adding this? Is it part of the spec? @timtebeek

- https://github.com/search?q=org%3Aopenrewrite%20annotation-api&type=code

Reply:
agree and please excuse.

- https://docs.openrewrite.org/recipes/java/migrate/javax/addcommonannotationsdependencies

Add the necessary annotation-api dependency from Jakarta EE 8 to maintain compatibility with Java version 11 or greater.